packages/core/utils/lib/convert-query-params.js

```diff
@@ -203,9 +220,15 @@ const convertPopulateObject = (populate, schema) => {
       return acc;
     }
 
+    const populateObject = convertNestedPopulate(subPopulate, targetSchema);
+
+    if (!populateObject) {
```

Comment:
Should we do a strict false check here ?

Reply:
Yes it's the same. It should be either an `object` or `true` or `false`.